packages/core/src/RenderingEngine/VideoViewport.ts

```diff
@@ -553,6 +553,10 @@ class VideoViewport extends Viewport {
       loop: false,
       muted: true,
```

Comment:
Other viewports, such as StackViewport and VolumeViewport, implement a `voiRange` in their `setProperties`. If we're resetting properties, we can use `this.setProperties({voiRange})` to align with those other implementations.